tests/main/snap-pack-integrity/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Verify that snap pack works with integrity data appended
+
+# TODO: add systems we know have veritysetup package available
+# so far a lot of distributions don't have veritysetup in path
+systems:
+  - ubuntu-1*
+  - ubuntu-2*
```

Comment:
please add debian-*

Reply:
Just an FYI for observers, for debian-* to pass we need to land https://github.com/snapcore/snapd/pull/12789

Reply:
The mentioned PR has landed, and this is rebased.